File: 프로그래머스/unrated/181830. 정사각형으로 만들기/정사각형으로 만들기.py
def solution(arr):
    r, c = len(arr), len(arr[0])
    if r > c:
        for i in range(r):
            arr[i] += ([0]*(r-c))
    elif c > r:
        for i in range(c-r):
            arr.append([0]*c)
    
    return arr

File: 프로그래머스/0/181830. 정사각형으로 만들기/정사각형으로 만들기.java
class Solution {
    public int[][] solution(int[][] arr) {
        int max = Math.max(arr.length, arr[0].length);
        int[][] result = new int[max][max];
        
        for(int i=0; i<arr.length; i++) {
            for(int j=0; j<arr[0].length; j++) {
                result[i][j] = arr[i][j];                
            }
        }
        return result;
    }
}